lte/gateway/c/core/oai/test/s1ap_task/mock_s1ap_op.h

```diff
@@ -0,0 +1,64 @@
+/**
+ * Copyright 2020 The Magma Authors.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+#include <iostream>
+#include <fstream>
+#include <string>
+#include <vector>
+
+#include "lte/gateway/c/core/oai/common/common_defs.h"
+
+#define STR_DECODE_TABLE \
```

Comment:
I would suggest adding a comment on top of these define statements to make it clear what is the usage for them (encoding s1ap task state)